server/model.go

```diff
@@ -27,7 +27,11 @@ func parseFromModel(ctx context.Context, name model.Name, fn func(api.ProgressRe
 			return nil, err
 		}
 
-		return parseFromModel(ctx, name, fn)
+		modelpath = ParseModelPath(name.DisplayLongest())
```

Comment:
Not totally necessary for this change, but ideally we should use `types/model` here and deprecate `server/modelpath.go`

Reply:
that's not in scope since it's require changes across the board. there are a few pr already that make those changes. examples https://github.com/ollama/ollama/pull/3713, https://github.com/ollama/ollama/pull/3715